Chinese/20170117/20170117.md

```diff
@@ -0,0 +1,2 @@
+zhizunbao hui jia guo nian le!
+[GitHub](https://github.com/CPAN-PRC/resources/wiki/My-first-Pull-Request)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.